CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 * [CHANGE] Deprecate specification of Prometheus rules via ConfigMaps in favor of `PrometheusRule` CRDs
```

Comment:
Shall we follow up with this on the release 0.22.0?

Reply:
Yeah I felt it was a little too early as the v0.20.0 release was just 3 weeks ago.

Reply:
True that.